cinema_service/settings.py

```diff
@@ -12,6 +12,8 @@
 
 from pathlib import Path
 
+from cinema.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
Consider grouping imports using parentheses if there are multiple imports from the same module. This is a suggestion for better code style and readability.